thoth/storages/graph/postgres.py

```diff
@@ -814,6 +814,49 @@ def get_solved_python_packages_all(
         os_version = normalize_os_version(os_name, os_version)
         return self.__class__.get_python_packages_all(**locals())
 
+    def get_solved_python_package_names_all(
```

Comment:
I think we have the same already: https://github.com/thoth-station/storages/blob/11fa9910270780ff02aa3be28d328e3410fc1b4f/thoth/storages/graph/postgres.py#L2961 but called `get_python_package_version_names_all` because `python_package_version` is synonym of `solved` for us.

Reply:
my bad if confusing, I realized we have mixed nomenclature, sometimes with solved, sometimes without

Reply:
Oh, thanks for pointing that. Makes life easier 👍🏻 

/close